documentation/Tutorials/WPF/SimpleLogging.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: Simple logging
+---
+[Contents](Contents) [Previous](App_Xaml) [Next](ExpensesLogbook)
+
+## Simple logging
+
+### Introduction
+
+Before we continue to create a real application from this skeleton, there is one little topic to touch. Caliburn.Micro uses some magic and especially if you just started, it can be hard to find out why your application is not working. To help you out, there is an extremely simple logging class that shows at least how Caliburn.Micro tries to find the View and this really can be helpful. So, you may add this as an optional step to your solution.
+
+### Create the logging class
+
+Create a new class named ``DebugLogger`` which implements the ILog interface.
```

Comment:
As far as I know, it uses the built in logging. It it is nothing more but a reporting tool. If you have better ideas, please give me a reference to where I can find more information. I discovered this at StackOverflow. For the moment I leave it as it is.

Reply:
Yes you are using the correct interface and have implemented it correctly but since its a `DebugLogger` there is already one such implemented in `Caliburn.Micro` [DebugLogger](https://github.com/Caliburn-Micro/Caliburn.Micro/blob/master/src/Caliburn.Micro.Core/DebugLog.cs).